.pre-commit-config.yaml

```diff
@@ -24,6 +24,7 @@ repos:
         # Attempts to load all xml files to verify syntax.
       - id: check-yaml
         # Attempts to load all yaml files to verify syntax.
+        exclude: ".*(.github.*)$"
```

Comment:
Was pre-commit upset with the new files? If so could you post what it said?

Reply:
Only some vague message about YAML key. I also don't think pre-commit should bother since GitHub checks anyway and it is obvious when the template is broken.